test/tests/test_revault.py

```diff
@@ -9,11 +9,14 @@
     "cpfp_descriptor&wsh(multi(1,xpub661MyMwAqRbcG9F1VX1rYev3ZsFRQ8XdJYk7rYLiaCfLt3WjVQ5o12SeJHPQEUCC8NmTWFmyejXrp3GWTFQNrXHL7iUXcoRo79VEnmKGYhc/*,xpub6CZFHPW1GiB8YgV7zGpeQDB6mMHZYPQyUaHrM1nMvKMgLxwok4xCtnzjuxQ3p1LHJUkz5i1Y7bRy5fmGrdg8UBVb39XdXNtWWd2wTsNd7T9/*))",
     "deposit_descriptor&wsh(multi(2,xpub661MyMwAqRbcEgaXJWejvvXw2PHvr6ZF5ybqahrCnptM1kTNirjVETa6Ba4oTJz3Uppghzow6pQaGzyPaUVeb7vSufEYRUVMyNKX9PyRxWC/*,xpub6DEzq5DNPx2rPiZJ7wvFhxRKUKDoV1GwjFmFdaxFfbsw9HsHyxc9usoRUMxqJaMrwoXh4apahsGEnjAS4cVCBDgqsx5Groww22AdHbgxVDg/*))",
     "unvault_descriptor&wsh(andor(multi(1,xpub661MyMwAqRbcG9F1VX1rYev3ZsFRQ8XdJYk7rYLiaCfLt3WjVQ5o12SeJHPQEUCC8NmTWFmyejXrp3GWTFQNrXHL7iUXcoRo79VEnmKGYhc/*,xpub6CZFHPW1GiB8YgV7zGpeQDB6mMHZYPQyUaHrM1nMvKMgLxwok4xCtnzjuxQ3p1LHJUkz5i1Y7bRy5fmGrdg8UBVb39XdXNtWWd2wTsNd7T9/*),and_v(v:multi(2,02abe475b199ec3d62fa576faee16a334fdb86ffb26dce75becebaaedf328ac3fe,030f64b922aee2fd597f104bc6cb3b670f1ca2c6c49b1071a1a6c010575d94fe5a),older(3)),thresh(2,pkh(xpub661MyMwAqRbcEgaXJWejvvXw2PHvr6ZF5ybqahrCnptM1kTNirjVETa6Ba4oTJz3Uppghzow6pQaGzyPaUVeb7vSufEYRUVMyNKX9PyRxWC/*),a:pkh(xpub6DEzq5DNPx2rPiZJ7wvFhxRKUKDoV1GwjFmFdaxFfbsw9HsHyxc9usoRUMxqJaMrwoXh4apahsGEnjAS4cVCBDgqsx5Groww22AdHbgxVDg/*))))#0v6kshkd",
+    # unvault_descriptor without cosigners and using tpub
+    "unvault_descriptor_without_cosigs&wsh(andor(thresh(1,pk(tpubD6NzVbkrYhZ4XcB3kRJVob8bmjMvA2zBuagidVzh7ASY5FyAEtq4nTzx9wHYu5XDQAg7vdFNiF6yX38kTCK8zjVVmFTiQR2YKAqZBTGjnoD/*)),older(10),thresh(2,pkh(tpubD6NzVbkrYhZ4WmzFjvQrp7sDa4ECUxTi9oby8K4FZkd3XCBtEdKwUiQyYJaxiJo5y42gyDWEczrFpozEjeLxMPxjf2WtkfcbpUdfvNnozWF/*),a:pkh(tpubD6NzVbkrYhZ4XyJXPpnkwCpTazWgerTFgXLtVehbPyoNKVFfPgXRcoxLGupEES1tSteVGsJon85AxEzGyWVSxm8LX8bdZsz87GWt585X2wf/*))))#u70luyrn",
```

Comment:
Tests are using mainnet xpubs here. Probably better to use testnet and tpubs, but for now let's use xpubs. I'll update it to use testnet later.
If you update f469-disco submodule to master test will pass. I fixed the thresh issue.

```suggestion
    # unvault_descriptor without cosigners and using xpub
    "unvault_descriptor_without_cosigs&wsh(andor(thresh(1,pk(xpub661MyMwAqRbcFozCJEKQbfoEScGGAeV8Mx6RGwwYmFgeKyJTAfyye2L7uuCmNaZycG9DK3jWB9Gvpre94LTu7J3CheiQitNJjDF9RMtT5Kt/*)),older(10),thresh(2,pkh(xpub661MyMwAqRbcEyoQHjRmcCXrEw8YVZxecB1fmm17Dqs9muXBAQUrLGk9JGWBBoqrB9VnMdzN5u2D8dVdLnViTxWSbRmb58xNEX3GARisZBx/*),a:pkh(xpub661MyMwAqRbcGB7fwdofjHV6FsR2fTxC8tkb96eT453UaCaxKTgLUNHW2sjShw4eez7afHnwF2F8G4VfaeeD5Kg3TXrKtMKtXJvUK3Jh3uE/*))))",
```

Reply:
Ok, thank you for the suggestion and the fix